src/main/java/pl/Lukasz/Calories/Service/MealsService.java

```diff
@@ -0,0 +1,42 @@
+package pl.Lukasz.Calories.Service;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.Lukasz.Calories.Entity.Meals;
+import pl.Lukasz.Calories.Repository.MealsRepository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class MealsService {
+
+    @Autowired
+    private MealsRepository repo;
+
+    public List<Meals> listAll(Integer user) {
+       return repo.findByUser(user);
+    }
+
+    public void save(Meals meals){
```

Comment:
formatting - please be consistent `) {` - Investigate IntelliJ autoformatting so you don't need to remember that